data-prepper-plugins/kinesis-source/src/main/java/org/opensearch/dataprepper/plugins/kinesis/extension/KinesisLeaseCoordinationTableConfig.java

```diff
@@ -0,0 +1,22 @@
+package org.opensearch.dataprepper.plugins.kinesis.extension;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NonNull;
+import software.amazon.awssdk.regions.Region;
+
+@Getter
+public class KinesisLeaseCoordinationTableConfig {
+
+    @JsonProperty("table_name")
+    @NonNull
+    private String tableName;
+
+    @JsonProperty("region")
+    @NonNull
```

Comment:
Does it mean we do not support global tables?

Reply:
This implementation is to support a single table per pipeline.